quarkus/config-api/src/main/java/org/keycloak/config/DatabaseOptions.java

```diff
@@ -82,4 +87,127 @@ public class DatabaseOptions {
             .defaultValue(100)
             .description("The maximum size of the connection pool.")
             .build();
+
+    public static final Option<Boolean> DB_MULTIPLE_DATASOURCES = new OptionBuilder<>("db-multiple-datasources", Boolean.class)
```

Comment:
Can you explain more how this would be used?

If I have "start --db-kind-x" will I also need to use "db-multiple-datasources=true" to specify any of the runtime properties?

Reply:
@shawkins I'm still thinking about this, as I wanted to have some mechanism to hide that high count of new options to not show in help. However, we have some build-time options for datasources, and the flag might be more like runtime options. 

But I think I'll remove it.

Reply:
Removed